services/index/src/api.js

```diff
@@ -103,6 +113,11 @@ builder.declare({
   method: 'post',
   route: '/namespaces/:namespace?',
   name: 'listNamespacesPost',
+  scopes: {
+    if: 'namespaceGiven',
+    then: 'index:list-namespaces-post:<namespace>',
+    else: 'index:list-namespaces-post',
```

Comment:
This is just a different way of calling the same method, so this can be `index:list-namespaces..`